engines/csvlint/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM linterhub/image-ruby
+LABEL maintainer="linterhub@repometric.com"
+RUN apk add --no-cache libcurl && \
+    apk add --update make \
```

Comment:
Why do you need update?

Reply:
csvlint needs `make`

Reply:
I doubt you need --update to get make.

Reply:
Yes, you are right, it worked without `--update`. Removing.